magit-todos.el

```diff
@@ -361,6 +373,13 @@ This can be toggled locally in Magit buffers with command
     (remove-hook 'magit-status-sections-hook #'magit-todos--insert-todos)
     (remove-hook 'magit-status-mode-hook #'magit-todos--add-to-status-buffer-kill-hook)))
 
+(define-minor-mode magit-todos-follow-mode
+  :init-value nil
+  :lighter " Follow" 
```

Comment:
I think we can omit a lighter.  Instead, let's use `message` to indicate when the mode is enabled or disabled.